plugins/plugin-chart-echarts/src/Timeseries/controlPanel.ts

```diff
@@ -291,7 +291,7 @@ const config: ControlPanelConfig = {
   },
   controlOverrides: {
     row_limit: {
-      default: 100,
+      default: 10000,
```

Comment:
This default value was too small - should default to at least 10000 for timeseries charts.